### DISCUSSION TITLE
Illuminate\Routing\ResponseFactory::file(): Argument #2 ($headers) must be of type array

### DISCUSSION THREAD

**Discussion by 1jason1** [livewire/livewire#7794] [7. Bugs (archived)]
### Livewire version

v3.4.0

### Laravel version

v10.42.0

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

Using vite in dev mode to reload edits.

Edit any class, vite reloads site which is a blank page no errors. 

Manually reload the site still no errors.

Have Ray open then the errors are seen in Ray.

Issue exists with v3.4.0 and v3.4.1

Installed Livewire v3.3.5, issue goes away. 

### Repository or Wirebox URL

NA

### Additional information

This the error I'm seeing in Ray.

```php
Illuminate\Routing\ResponseFactory::file(): Argument #2 ($headers) must be of type array, Illuminate\Http\Response given, called in /vendor/livewire/livewire/src/Drawer/Utils.php on line 57
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@1jason1 can you provide an example component that demonstrates the issue so we can investigate? Thanks!

**Comment by 1jason1** [comment]
It comes from all the components, I'm also using Filament v3.

I have the following code in the AppPanelProvider.

```php
public function register(): void
    {
        parent::register();

        FilamentView::registerRenderHook('panels::body.end', fn (): string => Blade::render("@vite('resources/js/app.js')"));
    }

**Comment by lightwalkernet** [comment]
I am having the same problem: Bugsnag come up with this:
Illuminate\Routing\ResponseFactory::file(): Argument #2 ($headers) must be of type array, Illuminate\Http\Response given

Issue exists with v3.4.0 and v3.4.1

Installed Livewire v3.3.5, issue goes away.

Seems there is an issue loading livewire.js. If I copy and past the url into a browser it loads fine but not from the application.

**Comment by mzakrzewski1985** [comment]
I have the same problem
Livewire 3.4.1

**Comment by mzakrzewski1985** [comment]
no repair? maybe everyone was sitting in a lower version.

maybe [https://benjamincrozat.com/livewire-js-404-not-found#by-bundling-livewire-into-your-javascript](url)

not testing

**Comment by zepfietje** [comment]
@joshhanley @calebporzio, this bug has probably been introduced in https://github.com/livewire/livewire/pull/6990 by @apreiml.

The [headers passed here](https://github.com/apreiml/livewire/blob/d39837910aa14754aa942b0d04787a959997703a/src/Drawer/Utils.php#L57) should be an array, while in reality a [response is returned](https://github.com/apreiml/livewire/blob/d39837910aa14754aa942b0d04787a959997703a/src/Drawer/Utils.php#L77) in certain conditions.

**Reply by mzakrzewski1985** [reply]
so we are waiting for updates ?

**Reply by zepfietje** [reply]
I've sent Caleb a direct message to make sure he's aware of this issue.

**Comment by mzakrzewski1985** [comment]
so we are waiting for updates ?

**Comment by apreiml** [comment]
can you check if the issue is solved by applying this PR: https://github.com/livewire/livewire/pull/7812

**Comment by calebporzio** [comment]
Thanks for the reports, I just reverted the offending PR for now: https://github.com/livewire/livewire/pull/7814

Will tag shortly
